.setup/install_system.sh

```diff
@@ -164,6 +169,11 @@ else
 fi
 
 if [ ${WORKER} == 1 ]; then
+    # Setting it up to allow SSH as root by default
+    mkdir -p -m 700 /root/.ssh
+    cp /home/vagrant/.ssh/authorized_keys /root/.ssh
+
+    sed -i -e "s/PermitRootLogin prohibit-password/PermitRootLogin yes/g" /etc/ssh/sshd_config
```

Comment:
These should be wrapped in a `if [ ${VAGRANT} == 1]; then` or else it'll break running this on regular machines we want to setup as a worker.

Reply:
I agree, I am just not sure what to do since I don't pass the vagrant flag to `install_system.sh` in `install_worker.sh` because I don't want any of the other vagrant setup to happen, just this special setup for the worker, which I understand wouldn't work for a regular machine. Should I just pass another flag like `--vagrant-worker` and then replace this with 
`if [ ${VAGRANT_WORKER} == 1]; then`? Seems like it might be a bit much though.

Reply:
I would suggest that we reuse the vagrant flag and for stuff that would then get triggered you add an additional `${WORKER} == 0` in addition to the existing vagrant check.

This would perfectly capture the logic of "we are provisioning a regular vagrant VM" vs "we are provisioning a vagrant worker" without introducing a third flag.